website/blog/2022-10-05-Harness-CI-Onboarding/harness-ci-onboarding.md

```diff
@@ -0,0 +1,259 @@
+---
+slug: harness-ci-onboarding
+title: Harness CI Onboarding - Beginners Guide
+authors: [dhrubajyoti]
+tags: [continuous-integration, harness-ci, beginners-guide, getting-started]
+---
+
+## Introduction
+
+This beginner guide aims to help learners learn about the basic components of Harness CI and develop an understanding of the DevOps ecosystem involved in the Software Development lifecycle. In this article we will learn about the basic features of Harness CIE and also get started by building the first basic sample pipeline.
+
+Engineering team is usually expected to deliver error-free code at high frequency. A fast and reliable CI/CD pipeline is a major part for implementation of that in a sustainable model over time. Harness Continuous Integration tool which is built with test intelligence, native secrets, fine-grained RBAC, and extensible governance  as one of the best solutions in the marketplace for automated pipelines. Automated pipelines remove user errors, provide feedback loops to developers and helps enable fast product iterations.
+
+**What is a pipeline?**
+
+A Pipeline is an end-to-end process that delivers a new version of your software. It can be considered to be a cyclical process that includes integration, delivery, operations, testing, deployment, real-time updates, and metrics monitoring.
+
+For example: A pipeline can use the CI module of Harness to build, test & push code and then also a CD module to deploy the artifact to the production environment.
+
+### Prerequisites
+
+
+- Github Account ([Check out the other source code managers here](https://docs.harness.io/article/kqik8km5eb-source-code-manager-settings#source-code-manager-settings))
+- Docker Hub Account & Repository ([Other artifactory packages](https://docs.harness.io/article/66ykcm0sf0-build-and-push-to-gcr-step-settings))
+- Installation of K8s Delegate for Harness Delegate
+- About Harness CI
+- Features in Harness CI
+- Getting Started with your first pipeline
+- Troubleshooting frequently encountered issues
+- Conclusion: Developer Feedback on Harness CIE
+- What’s Next?
+
```

Comment:
Add `### Environment`
- Ubuntu Version

Reply:
The `Environment` section is missing! Could you please add the OS env details after `Prerequisites`?
Quick reference [here](https://github.com/Ompragash/deploy-galaxyng/blob/main/docs/galaxyng_server_install.md)